### DISCUSSION TITLE
Разрешите токарю смотреть, есть лицензия у человека или нет.

### DISCUSSION THREAD

**Discussion by SemjonBelus** [octothorp-team/public-dobrograd#701] [Предложения]
Я являюсь одним из работников G&A и нам нужно проверять лицензию на оружие, но иногда у человека её может просто не быть и он её "придумывает". Да, это нарушает правила, но токарь об этом даже не может узнать. Сегодня я помогал с этой проблемой моему другу и подобные ситуации имели место быть, хоть и не часто.
Я хотел бы, чтобы люди с профессией токаря имели возможность смотреть через TAB лицензии людей.

**Comment by Emberenius** [comment]
Я считаю, что это вовсе не нужно, так как в США у тебя в магазине точно не попросят предъявить лицензию. Там с оружием все обстоит намного проще и ты можешь купить, к примеру, ружье с помощью водительского удостоверения, подтверждающего твое 21-летие. Я твой папищек кстати

**Comment by Kyper1** [comment]
Конкретно на Доброграде, по стандарту, если у тебя нет лицензии на оружие, то оно - нелегальное.

**Comment by travka4000** [comment]
не знаю что ответить так-что я отношусь к этому 50/50
